src/components/tree/interfaces.ts

```diff
@@ -8,5 +8,6 @@ export enum TreeSelectionMode {
   None = "none",
   Children = "children",
   MultiChildren = "multi-children",
```

Comment:
Does this need to be renamed too?

Reply:
I think an abbreviated name is more suitable here.

Reply:
`multi` is a combining form word, so let's change this to `multichildren`.